Xamarin.Forms.Material.iOS/MaterialEntryRenderer.cs

```diff
@@ -11,6 +12,11 @@ public class MaterialEntryRenderer : EntryRendererBase<MaterialTextField>, IMate
 		protected override void UpdateColor() => Control?.UpdateTextColor(this);
 		protected virtual void ApplyTheme() => Control?.ApplyTheme(this);
 		protected override void UpdatePlaceholder() => Control?.UpdatePlaceholder(this);
+		protected override void UpdateAttributedPlaceHolder(NSAttributedString nSAttributedString)
+		{
+			// Attributed place holders don't currently work with Material Text Fields
+			// once/if it does start working it will be handled inside MaterialTextManager	
```

Comment:
once/if _they do_ start working _they_ will be handled inside MaterialTextManager